screen_brightness_control/__init__.py

```diff
@@ -503,14 +509,14 @@ def get_brightness(self) -> IntPercentage:
     def get_identifier(self) -> Tuple[str, DisplayIdentifier]:
         '''
         Returns the `.types.DisplayIdentifier` for this display.
-        Will iterate through the EDID, serial, name and index and return the first
+        Will iterate through the EDID, serial, name, uid and index and return the first
         value that is not equal to None
 
         Returns:
             The name of the property returned and the value of said property.
             EG: `('serial', '123abc...')` or `('name', 'BenQ GL2450H')`
         '''
-        for key in ('edid', 'serial', 'name'):
+        for key in ('edid', 'serial', 'name', 'uid'):
```

Comment:
Since UID appears to be the most unique, especially in the case of duplicate EDIDs, maybe it should take priority in this list and be the first thing we filter on/use as an identifier.